docs/source/standard.md

```diff
@@ -223,33 +223,87 @@ The ability of the TRE organisation to manage data assets and ensure information
 
 The ability of the TRE organisation to manage how and where data is stored, how it moves, changes and is removed.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+```{list-table}
+:header-rows: 1
+:name: tab-data-lifecycle-management
+
+* - Statement
+  - Guidance
+* - A TRE must have a data ingress process which enforces information governance rules/processes.
+  - The data ingress process needs to ensure that information governance is correctly followed.
+    in particular, it should require that an ingress request has been approved by all required par ties.
+* - A TRE must have a data egress process which enforces information governance rules/processes.
+  - The data egress process needs to ensure that information governance requirements are adhered to.
+    In particular, it should require that an egress request has been approved by all required parties.
+* - A TRE could require independent approval for data ingress and egress events.
+  - A person independent to a project, for example a TRE admin or dedicated referee, could check ingress and egress requests to ensure information governance processes are correctly followed in the absence of an automated process.
+* - A TRE must keep a record of what data it holds.
+  - Good records are important for ensuring compliance with legislation, understanding risk and aiding good data hygiene.
+    The record should include a description of the data, its source, contact details for the data owner, which projects use the data, the data it was recieved, when it is expected to no longer be needed.
+* - A TRE must have a policy on data deletion.
+  - There should be a clear, published policy on when data will be deleted beyond at the conclusion of a project.
+    This may allow time for data owners to consider outputs they may want to extract from the TRE.
+    Any sensitive data, including all backups, should be deleted when they are no longer needed.
+    Having clear policy will help to avoid problems with data being kept longer than necessary or accidental deletion of outputs.
```

Comment:
```suggestion
    Having clear policies will help to avoid problems with data being kept longer than necessary or accidental deletion of outputs.
```
or `a clear policy`?